var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -60,6 +61,7 @@
     "79580508b039ca6c50dfdfd7c4f6fbcf489fe1931037ca51324818851eea0c1c",
     "e9c2481cccacdea72c1f8d3970956c447cec47e18dfb9712cbbba76a2820552c",
     "1f52e45660ea508d3fe717a9903fe27020cee96de95a3541434838e0193a4827",
+    "",
```

Comment:
It's the same as llvm-amdgpu
```suggestion
    "4df9aba24e574edf23844c0d2d9dda112811db5c2b08c9428604a21b819eb23d",
```